lib/oauthutil/oauthutil.go

```diff
@@ -269,6 +269,23 @@ func (ts *TokenSource) Invalidate() {
 	ts.mu.Unlock()
 }
 
+// Expire marks the token as expired
+//
+// This also marks the token in the config file as expired, if it is the same one
+func (ts *TokenSource) Expire() error {
```

Comment:
Currently there is no way for backends to detect invalid access tokens and request a new one by invalidating the old access token (but keeping the refresh token).

Of course this change is not strictly needed, as invalid access tokens can not be generated by valid refresh tokens, thus such a situation can not be caused by rclone itself, at least regarding HiDrive.
This is more or less a safeguard against accidental changes to the access-token when editing the configuration.

Reply:
This may be useful in situations where an access token can be revoked early.

Reply:
We don't know details of your API so we defer to your authority.
rclone has been widely used in automated workflows so the more stable is the code, the better.